src/App.jsx

```diff
@@ -1,31 +1,52 @@
-import React, { useState, useEffect } from 'react';
-import { getAllData } from './util/index';
+import React, { useState, useEffect } from "react";
+import { getAllData } from "./util/index";
+import Header from "./components/Header/Header.jsx";
+import RecipePage from "./components/RecipePage/Recipepage.jsx";
```

Comment:
Quick fix: import RecipePage from "./components/RecipePage/RecipePage.jsx" (capitalize the P in Recipepage)